lib/galaxy/managers/jobs.py

```diff
@@ -0,0 +1,161 @@
+import json
+from six import string_types
+from sqlalchemy import and_, or_
+from sqlalchemy.orm import aliased
+
+from galaxy import model
+from galaxy.managers.hdas import HDAManager
+from galaxy.managers.collections import DatasetCollectionManager
```

Comment:
Wrong import order (you may want to add this file to `.ci/flake8_lint_include_list.txt` so it will be checked)